spec/support/factory_girl.rb

```diff
@@ -5,6 +5,10 @@
     sequence(:email) {|n| "user#{n}@example.com" }
     password 'password'
     password_confirmation 'password'
+    role 'user'
+    factory :user_admin do
+      role 'admin'
+    end
```

Comment:
Kill the factory :user_admin; you can adjust `role` with what you want in your factory in your test (like I did in the examples above).